docs/track.rst

```diff
@@ -883,8 +883,13 @@ Properties
 * ``body`` (mandatory): The query body.
 * ``response-compression-enabled`` (optional, defaults to ``true``): Allows to disable HTTP compression of responses. As these responses are sometimes large and decompression may be a bottleneck on the client, it is possible to turn off response compression.
 * ``detailed-results`` (optional, defaults to ``false``): Records more detailed meta-data about queries. As it analyzes the corresponding response in more detail, this might incur additional overhead which can skew measurement results. This flag is ineffective for scroll queries.
-* ``pages`` (optional): Number of pages to retrieve. If this parameter is present, a scroll query will be executed. If you want to retrieve all result pages, use the value "all".
-* ``results-per-page`` (optional):  Number of documents to retrieve per page for scroll queries.
+* ``results-per-page`` (optional): Number of results to retrieve per page.  This maps to the Search API's ``size`` parameter, and can be used for paginated and non-paginated searches.  Defaults to ``10``
```

Comment:
should this just be called `size`?  there didn't seem to be a good reason to make this only for scroll/pagination given it maps to something consequential to the single request_body search

Reply:
can/should i deprecate `results-per-page` here?